server.js

```diff
@@ -7517,6 +7517,70 @@ function(data, match, end, ask) {
   }
 });
 
+// Distelli integration
+camp.route(/^\/distelli\/([^/]+)\/([^/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/, cache({
+  queryParams: ['token'],
+  handler: (data, match, sendBadge, request) => {
+    const type = match[1]; // build or deploy
+    const org = match[2];
+    const app = match[3];
+    const format = match[4];
+    const token = data.token;
+    const badgeData = getBadgeData(type, data);
+    const apiType = type === 'build' ? 'builds' : 'deployments';
+    const apiUrl = 'https://api.distelli.com/' + org + '/apps/' + app + '/' + apiType + '?order=desc&max_results=1&apiToken=' + token;
+
+    const statusColorScheme = {
+      unknown: 'lightgrey',
+      waiting: 'lightgrey',
+      queued: 'lightgrey',
+      running: 'lightgrey',
+      success: 'brightgreen',
+      failed: 'red'
+    };
+
+    request(apiUrl, {json: true}, function(err, res, data) {
+      try {
+        if (!res || err !== null || res.statusCode !== 200) {
+          badgeData.text[1] = 'inaccessible';
+          sendBadge(format, badgeData);
+          return;
+        }
+
+        const apiStatus = type === 'build' ? data.builds[0] : data.deployments[0].servers;
+        let status = 'unknown';
+
+        switch (type) {
+          case 'build':
+            status = apiStatus.status.toLowerCase();
+            break;
+          case 'deploy':
+            var hasErrors = apiStatus.failed > 0 ? true : false;
+            if (hasErrors) {
+              status = 'failed';
+            } else if (apiStatus.in_progress > 0 || apiStatus.pending > 0 || apiStatus.waiting > 0) {
+              status = 'running';
+            } else if (apiStatus.done > 0) {
+              status = 'success';
+            }
+            break;
+          default:
+            break;
+        }
+
+        const message = status;
```

Comment:
This line seems unnecessary. ^^

Reply:
It's certainly is, not sure why I didn't remove it.